core/util/src/main/scala/net/liftweb/util/TimeHelpers.scala

```diff
@@ -54,96 +51,86 @@ trait TimeHelpers { self: ControlHelpers =>
   /** transforms an int to a TimeSpan object. Usage: 3000 returns a TimeSpan of 3000L millis  */
   implicit def intToTimeSpan(in: Int): TimeSpan = TimeSpan(in)
 
-  private implicit def durToPeriod(dur: Duration): Period = dur.toPeriod(PeriodType.standard())
-
   /** class building TimeSpans given an amount (len) and a method specify the time unit  */
   case class TimeSpanBuilder(val len: Long) {
-    def seconds = new TimeSpan(Right((new Period).plusSeconds(len.toInt)))
+    def seconds = new TimeSpan(Left(Duration.standardSeconds(len)))
     def second = seconds
-    def minutes = new TimeSpan(Right((new Period).plusMinutes(len.toInt)))
+    def minutes = new TimeSpan(Left(Duration.standardMinutes(len)))
     def minute = minutes
-    def hours = new TimeSpan(Right(Duration.standardHours(len): Period))
+    def hours = new TimeSpan(Left(Duration.standardHours(len)))
     def hour = hours
-    def days = new TimeSpan(Right(Duration.standardDays(len): Period))
+    def days = new TimeSpan(Left(Duration.standardDays(len)))
     def day = days
-    def weeks = new TimeSpan(Right(Duration.standardDays(len * 7L): Period))
+    def weeks = new TimeSpan(Left(Duration.standardDays(len * 7L)))
     def week = weeks
-    def months = new TimeSpan(Right((new Period().plusMonths(len.toInt))))
+    @deprecated("TimeSpan will not support operations on non-millis periods in future")
```

Comment:
Let's use 3.0.0. I think it's worth going full semver for the 3 series, because why not :)